settings.gradle

```diff
@@ -285,6 +285,9 @@ project(':docker-web-plugin-packager').projectDir = file('docker/web-plugin-pack
 include(':docker-server-slim')
 project(':docker-server-slim').projectDir = file('docker/server-slim')
 
+include(':hotspot')
```

Comment:
I'm wondering if we want a more specific package name (I choose "hotspot" without much thought). `hotspot` seems to imply more of a library, rather than an implementation. We may want to extract `HotSpot` interface outside of `engine-util` in the future(or now?), and `hotspot` might be the best name for that. For the implementation, maybe `hotspot-impl`?